docs/commands/got.md

```diff
@@ -6,9 +6,11 @@ The `got` command optionally takes function names and filters the output display
 matching functions.
 
 ```text
-gef➤ got
+gef➤ got [--all]
 ```
 
+'--all' Print the GOT for all shared objects in addition to the executable file
+
 ![gef-got](https://i.imgur.com/554ebM3.png)
 
 The applied filter partially matches the name of the functions, so you can do something like this.
```

Comment:
please wrap like the rest

Reply:
Can you clarify what "like the rest" means?  markdownlint does have some findings for pre-existing content in this file, but not for the lines that I've added -- that appears to be shorter than the maximum line length.

Reply:
Right, I'm saying you should wrap your lines to match, rather than doing 80 when I think we allow 100 or 120.

Reply:
Oh, the github review makes it look like you were referring to line 12, but I think you're actually referring to line 38.  I've un-wrapped that line.